app/http/httpclient.yaml

```diff
@@ -33,7 +33,7 @@ components:
           x-field-uid: 3
         cookie_reject_probability:
           description: >-
-            cookie_reject_probability indicates the probability, from 0 to 1, that a client will reject a request for a cookie’s contents from the server. (only if enable_cookie_support is enabled)
+            cookie_reject_probability indicates the probability, from 0 to 1, that a client will reject a request for a cookie(s) contents from the server. (only if enable_cookie_support is enabled)
           type: integer
```

Comment:
Should this be an integer or some other type?

Reply:
updated with boolean